content/en/docs/contribute/style/hugo-shortcodes/index.md

```diff
@@ -49,6 +49,21 @@ Renders to:
 
 {{< feature-state for_k8s_version="v1.10" state="beta" >}}
 
+### Feature state  retrieval from description file
```

Comment:
nit: spacing
```suggestion
### Feature state retrieval from description file
```

Reply:
Updated as part of latest commit. Thanks for review.